### PR TITLE
implement install command (closes #5)

### LINKED ISSUE #5: install command
`install` command installs Base OS on a board.

## Usage
```
Usage: codestand install device-name board serial
```

## Example
```
$ codestand install my-board esp8266 /dev/tty.USB0
```

### PR DESCRIPTION
**Pull request by seiyanuta**

### CODE REVIEW

File: install.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	Dev:   false,
+	Run:   runInstall,
+	Usage: "install",
+	Short: "install OS",
+}
+
+
+var board string
+var serial string
+var image string
```
Comment (sh19910711) [inline install.go]: Can you put a prefix like `installBoard` to the global variable names? It could avoid to conflict with other commands, perhaps.

Comment (seiyanuta) [inline install.go]: The root cause is separating command line parsing and command stuffs such as installing a firmware. How about put them together in another PR?

Comment (sh19910711) [inline install.go]: Sounds good.
